Makefile

```diff
@@ -70,5 +70,9 @@ rye-build: rye-src
 	export GO111MODULE=${GO111MODULE} && \
 		cd rye-src && go build -x -tags "b_tiny" -o rye .
 
+.PHONY: haskell
```

Comment:
btw I think most of the Make targets should be set as PHONY.

Reply:
This is correct. I've patched Makefile on the `master` to address this. Tnx!